src/DIRAC/Core/Utilities/ExecutorDispatcher.py

```diff
@@ -836,18 +832,12 @@ def __sendTaskToExecutor(self, eId, eTypes=False, checkIdle=False):
         return S_OK(taskId)
 
     def __msgTaskToExecutor(self, taskId, eId, eType):
-        try:
-            self.__tasks[taskId].sendTime = time.time()
-        except KeyError:
```

Comment:
Is it really OK to drop this check?

Reply:
It's only re-raising the exception and all expceptions are caught by `__sendTaskToExecutor` a few lines higher.